mava/configs/default/rec_iql.yaml

```diff
@@ -4,7 +4,7 @@ defaults:
   - arch: anakin
   - system: q_learning/rec_iql
   - network: rnn  # [rnn, rcnn]
-  - env: smax  # [cleaner, connector, gigastep, lbf, matrax, rware, smax]
+  - env: smax  # [cleaner, connector, gigastep, lbf, matrax, rware, smax, mpe]
```

Comment:
Can we only add it to cont systems, I know it can be discrete, but lets treat it as one of our continuous envs

Reply:
Ah right the default action space is set to continuous 😅